File: IntelliJ IDEA/Java/src/demo/itemsControlImplementCustomViewList/Item.java
package demo.itemsControlImplementCustomViewList;

public class Item {

	int id;

	public Item (int id) {
		this.id = id;
	}

	public int getId () {
		System.out.println ("get id: " + id);
		return id;
	}

	public void setId (int id) {
		this.id = id;
	}

}

File: Unity/Assets/Eruru.MVVM/Demos/ItemsControl/Item.cs
using UnityEngine;
using System.Collections;

namespace Eruru.MVVM.Demo.ItemsControl {

	public class Item {

		public string Name { get; set; }
		public int Age { get; set; }
		public string School { get; set; }
		public string Remark { get; set; }

		public Item () {

		}
		public Item (string name, int age, string school, string remark) {
			Name = name;
			Age = age;
			School = school;
			Remark = remark;
		}

		public Item Clone () {
			return new Item (Name, Age, School, Remark);
		}

	}

}